core/store/ledgerstore/ledger_store.go

```diff
@@ -173,18 +173,20 @@ func (this *LedgerStoreImp) InitLedgerStoreWithGenesisBlock(genesisBlock *types.
 		genHash := genesisBlock.Hash()
 		log.Infof("GenesisBlock init success. GenesisBlock hash:%s\n", genHash.ToHexString())
 	} else {
-		genesisHash := genesisBlock.Hash()
-		exist, err := this.blockStore.ContainBlock(genesisHash)
-		if err != nil {
-			return fmt.Errorf("HashBlockExist error %s", err)
-		}
-		if !exist {
-			return fmt.Errorf("GenesisBlock arenot init correctly, hash %s", genesisHash.ToHexString())
-		}
 		err = this.init()
 		if err != nil {
 			return fmt.Errorf("init error %s", err)
 		}
+		if this.currBlockHeight == 0 {
```

Comment:
remove this if?
the following checks are always needed.

Reply:
This is a temporary change, which will be optimized later